travellan/travellan-rn
# Bugfix/367 transport redux

## Changes made

- I refactored transport in Redux:
  - I created action creators,
  - I refactored thunks,
- I cleaned up containers and components,
- I fixed an error where when creating a trip, its Firebase key is undefined.

## Screenshots

(prefer animated gif)

## Test path (if necessary)

1. Select a trip.
2. Go to Transport screen (there is no error now).
3. Go to Add transport screen and add transport. You go back to Transport screen and can see the created Transport.
4. Delete some transport. You should see that it disappears.

## Checklist

- [ ] rebased feature branch with develop branch
  - [x] in emulator
  - [ ] on a real device
- [ ] added new dependencies (what deps?)
- [x] used ESLint (linted) and Prettier (formatted)
